### ISSUE TITLE
[Feature Request] Hide 'Explain this post' when 'Hide Grok' is enabled

### ISSUE THREAD

**Issue by GlitchTheFox** [insin/control-panel-for-twitter#669]
![image](https://github.com/user-attachments/assets/50a33a0e-56aa-4aec-b94a-2ae4e4529863)
![image](https://github.com/user-attachments/assets/d09e7301-72b0-4904-9bf3-7640dae9ead1)

A new tumour has developed on the cancer-ridden husk of Twitter.
I'd very much appreciate if this can be hidden when the 'Hide Grok' option is enabled. I'm using the Firefox extension, so if this is already implemented on other versions of the extension, then pls ignore.

**Comment by nokel** [comment]
I too would like this!

**Comment by PinballsWizard** [comment]
noticed the same here

**Comment by jadeposting** [comment]
needs to be expanded to hide these new prompts on unfocused tweets ([example tweet](https://x.com/charlieINTEL/status/1876633118129484188)]
![image](https://github.com/user-attachments/assets/8fd75c82-a8af-4206-b3f2-db7aa2d5ed87)

**Comment by nokel** [comment]
Yeah, I started seeing these as well except for some reason the first  question box always has all the individual letters of the sentence going downwards... not sure why though... Could potentially be something to do with another addon if no one else is having the same issue...

**Comment by paketep** [comment]
I have the same problem. Would love to be rid of this sh*t.

**Comment by Hok-2323** [comment]
i think just need to add
```'div:has([data-testid^="followups_"])'```
since the structure is like
```
<tweet content> 
<div>
    <div data-testid="followups_[]"> </div>
    <div>
    [grok respones]
    </div>
</div>
```

**Comment by nokel** [comment]
There's a new addition to this, the bottom right now has a popup.

![Image](https://github.com/user-attachments/assets/83b809a3-0cce-4ec1-a340-dd131457b34c)
